### PR TITLE
Fix restart command in FLARE API

### PR DESCRIPTION
**Pull request by nvkevlu**
Fixes restart command in FLARE API.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/fuel/flare_api/flare_api.py [hunk 1]
```diff
@@ -470,7 +470,7 @@ def restart(self, target_type: str, client_names: Optional[List[str]] = None) ->
         if target_type not in _VALID_TARGET_TYPES:
             raise ValueError(f"invalid target_type {target_type} - must be in {_VALID_TARGET_TYPES}")
 
-        parts = [AdminCommandNames.CHECK_STATUS, target_type]
+        parts = [AdminCommandNames.RESTART, target_type]
```
Comment (YuanTingHsieh) [inline nvflare/fuel/flare_api/flare_api.py:473]: This line LGTM.

Can you help just correct the docstring style to follow other methods? (basically the method description should follow """ directly.) thanks

Comment (nvkevlu) [reply nvflare/fuel/flare_api/flare_api.py:473]: I looked and realized there's a bunch of places docstrings can use cleaning up in this file. Is it okay if I put this change with the other changes in a separate PR tomorrow? Or I suppose if this is still open I can add it here.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks for the docstring fixes, LGTM.

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**